packages/vector_graphics_compiler/lib/src/svg/tesselator.dart

```diff
@@ -0,0 +1,377 @@
+// ignore_for_file: camel_case_types
```

Comment:
This file should probably be sourced from the engine, we should just upload it along side libtessellator. No change needed for this PR.

Reply:
How are we going to import a dart file we don't know the location of until we run the program? Unless you mean we should roll it in?

Reply:
Maybe it should get published to pub and contain the routine for finding the libtessellator in the flutter cache itself?

Reply:
Alternatively it's just a pub package in the cache and we figure out some weird way to trick pub into pulling it from the SDK cache haha

Reply:
I think you should figure out what you want to do here and whether that can work independent of this PR

Reply:
Yes, "not for this PR" :)

Reply:
I missed that :P